ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -458,7 +590,7 @@ private class SegmentedControlButton: UIButton {
 
         super.init(frame: .zero)
 
-        contentEdgeInsets = style == .switch ? Constants.contentEdgeInsetsForSwitch : Constants.contentEdgeInsets
+        contentEdgeInsets = Constants.contentEdgeInsets
```

Comment:
we propbably don't need multipe segmentcontrol.style anymore right? and also we can delete contentedgeinsetforswitch static let ?